pkg/infra/game/decision/decision.go

```diff
@@ -6,19 +6,22 @@ import (
 	"github.com/benbjohnson/immutable"
 )
 
-type ProposalAction interface {
-	FightAction | LootAction
-}
-
 type HPPoolDecision struct{}
 
 type Manifesto struct {
+	running            bool
 	fightDecisionPower bool
 	lootDecisionPower  bool
+	fightPolicy        []Rule[FightAction]
```

Comment:
This should be immutable lists to avoid leaders being able to change their manifesto